CHANGELOG.md

```diff
@@ -14,3 +14,41 @@ All notable changes to this project will be documented in this file. The format
   - AMKO support for `GSLBHostRule`.
   - AMKO support for GslbService properties: TTL, Site Persistence and Custom Health Monitors via `GlobalDeploymentPolicy` and `GSLBHostRule`.
   - AMKO support for adding third party site members via `GSLBHostRule`.
+
+## AMKO-1.4.2
+
+### Added:
+  - Support for [federation](docs/AMKO/federation.md). It has to be enabled during installation.
+  - A new custom resource `AMKOCluster` for federation configuration.
+  - AMKO can now boot up, even if one of the member clusters is unreachable. If the cluster is available later on, AMKO will start it's informers.
+
+### Bugs fixed:
+  - Parsing error for TTL and hash mask fields after creation of GslbServices
+  - Path based health monitors gets unnecessarily created and then deleted sometimes, even if custom health monitor refs are provided
+  - GslbServices unnecessarily updated due to incorrect parsing of site persistence field
+  - AMKO doesn't panic if the GSLB leader details couldn't be fetched
+
+## AMKO-1.5.1
+
+### Added:
+  - Support for rebooting AMKO if the GSLB leader IP address is updated in the `GSLBConfig` object.
+
+### Bugs fixed:
+  - Fixed a status update race between re-sync interval goroutine and modification to `GSLBConfig` object.
+
+## AMKO-1.5.2
+
+### Updated:
+  - Base image `photon:4.0` updated with fixes for latest vulnerabilities
+
+## AMKO-1.6.1
+
+### Added:
+  - AMKO now claims support for Kubernetes 1.22.
```

Comment:
I think we can remove claims and just say that AMKO supports 1.22

Reply:
I have addressed the comment.